### PR TITLE
deal with extra slashes in, i.e. , dropbox:////some/place #396

### PR DESCRIPTION
**Pull request by marcmengel**
Small tweak in fake_ifdh.py to keep it from getting confused by extra slashes, or passing them on to gfal-copy and confusing it.   Does not mess with other URL types, etc.

### CODE REVIEW

File: lib/fake_ifdh.py [hunk 1]
```diff
@@ -234,6 +234,10 @@ def generate_proxy_command_verbose_args(cmd_str: str) -> Dict[str, Any]:
 
 
 def fix_pnfs(path: str) -> str:
+
```
Comment (shreyb) [inline lib/fake_ifdh.py:237]: Would it be possible to have a unit test or two added to make sure this works as we expect (I mean, it SHOULD, but...)

**Review by shreyb — CHANGES_REQUESTED** [review]
Code itself is good.  Requested unit test.

File: lib/fake_ifdh.py (hunk 1 above)
Comment (marcmengel) [reply lib/fake_ifdh.py:237]: On it...

Comment (marcmengel) [reply lib/fake_ifdh.py:237]: Added.

Comment (shreyb) [reply lib/fake_ifdh.py:237]: Thanks!

**Review by shreyb — APPROVED** [review]
LGTM.

**Pull request merged**